test/components/guideLineLayerTests.ts

```diff
@@ -1,481 +1,471 @@
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let gll = new Plottable.Components.GuideLineLayer<number>("vertical");
-      let value = 5;
-      gll.value(value);
-      gll.renderTo(svg);
-
-      let scale1 = new Plottable.Scales.Linear();
-      scale1.domain([0, 10]);
-      gll.scale(scale1);
-      let line = gll.content().select(".guide-line");
-      let expectedAttrs1 = {
-        x1: scale1.scale(value),
-        x2: scale1.scale(value),
-        y1: 0,
-        y2: SVG_HEIGHT
-      };
-      TestMethods.assertLineAttrs(line, expectedAttrs1, "the line was drawn at the correct position");
-
-      scale1.domain([0, 20]);
-      let expectedAttrs1b = {
-        x1: scale1.scale(value),
-        x2: scale1.scale(value),
-        y1: 0,
-        y2: SVG_HEIGHT
-      };
-      TestMethods.assertLineAttrs(line, expectedAttrs1b, "the line was redrawn at the correct position on domain change");
-
-      let scale2 = new Plottable.Scales.Linear();
-      scale2.domain([0, 100]);
-      gll.scale(scale2);
-      line = gll.content().select(".guide-line");
-      let expectedAttrs2 = {
-        x1: scale2.scale(value),
-        x2: scale2.scale(value),
-        y1: 0,
-        y2: SVG_HEIGHT
-      };
-      TestMethods.assertLineAttrs(line, expectedAttrs2, "the line was redrawn at the new position when the scale was changed");
-
-      svg.remove();
-    });
-
-    it("sets the scale's range based on the allocated width", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let gll = new Plottable.Components.GuideLineLayer<number>("vertical");
-      let scale1 = new Plottable.Scales.Linear();
-      gll.scale(scale1);
-      gll.renderTo(svg);
-      assert.deepEqual(gll.scale().range(), [0, SVG_WIDTH], "range was set based on the allocated width");
-
-      let scale2 = new Plottable.Scales.Linear();
-      gll.scale(scale2);
-      assert.deepEqual(gll.scale().range(), [0, SVG_WIDTH], "replacement scale has its range set based on the allocated width");
-
-      svg.remove();
-    });
-  });
-
-  describe("rendering (horizontal)", () => {
-    let SVG_WIDTH = 300;
-    let SVG_HEIGHT = 400;
-
-    it("requests no space, but will occupy all offered space", () => {
-      let gll = new Plottable.Components.GuideLineLayer<void>("horizontal");
-      let request = gll.requestedSpace(SVG_WIDTH, SVG_HEIGHT);
-      TestMethods.verifySpaceRequest(request, 0, 0, "does not request any space");
-      assert.isTrue(gll.fixedWidth(), "fixed width");
-      assert.isTrue(gll.fixedHeight(), "fixed height");
-
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      gll.anchor(svg);
-      gll.computeLayout({x: 0, y: 0}, SVG_WIDTH, SVG_HEIGHT);
-      assert.strictEqual(gll.width(), SVG_WIDTH, "accepted all offered width");
-      assert.strictEqual(gll.height(), SVG_HEIGHT, "accepted all offered height");
-      svg.remove();
+describe("Interactive Components", () => {
```

Comment:
`GuideLineLayer` isn't interactive, since it doesn't include an `Interaction`, so I don't think it belongs in this `describe()`.